contents/docs/feature-flags/bootstrapping.mdx

```diff
@@ -74,9 +74,9 @@ The `bootstrap` object has two optional arguments: `distinctID` and `isIdentifie
 
 - `distinctID` enables you to set distinct ID of the user during PostHog's initialization. This is useful when you want to ensure the distinct ID on your frontend is the same as the distinct ID that you called `getAllFlags()` with on your server. It is **strongly recommended** to include it.
 
-> **Note:** The only case where it doesn't make sense to include the distinctID is when you're bootstrapping identified users only, and you call identify for these users right on load. Here, if you pass in the distinctID to bootstrap, identify will not connect your existing anonymous ID, since it's replaced by the bootstrapped ID.
+> **Note:** The only case where it doesn't make sense to include `distinctID` is when your code only runs for [identified users](/docs/product-analytics/identify) **and** you call `identify` immediately on load. 
```

Comment:
@neilkakkar is this accurate?

Reply:
No, the idea is you call `identify()` after bootstrapping to link anonymous users to this logged in user.

But, if you bootstrap the logged in user's distinct ID, then the identify() does nothing, and you lose the link to the anonymous user

Reply:
@neilkakkar 

> But, if you bootstrap the logged in user's distinct ID, then the identify() does nothing, and you lose the link to the anonymous user

Right, but you wouldn't need the anonymous user ID if you've bootstrapped. So yes it doesnt make sense to call identify if you're bootstrapping an ID. But it wont affect you negatively/break anything

In this case, can I just remove this note?

Reply:
Hmm, let me try explaining the case we want to highlight, and then you can edit this to say it 😅 -> This is a true case that applies also to us ->

1. I have an anonymous user on posthog.com browsing
2. They now go to the app and login, and we bootstrap after login. At this stage, we know the user's identity.

We _never_ pass in the distinct id to bootstrap in (2), even if we know it, and instead call `identify()` because we want to connect the anonymous user in (1), to the identified user in (2). If instead we pass the bootstrapped identified id in, and don't call identify, we lose this connection and lose data for the anon user.